src/test/cypress/support/pageobjects/exercises/programming/OnlineEditorPage.ts

```diff
@@ -200,6 +185,7 @@ export class OnlineEditorPage {
 export class ProgrammingExerciseSubmission {
     files: ProgrammingExerciseFile[];
     expectedResult: string;
+    packageName: string;
```

Comment:
There are several changes concerning the `packageName`. I needed more time to read it all in depth and am not really into Cypress Tests. What is being changed concerning those in general in this PR?

Reply:
I moved the definition of the `packageName` from the cypress test itself to the submissions file.